Google.Api.Generator.Tests/ProtoTest.cs

```diff
@@ -92,18 +92,24 @@ public void ProtocExecution()
             }
         }
 
-        private void BuildTest(string testName, bool ignoreUnitTests = false)
+        private void BuildTest(string serviceName, string protoPackageVersion = null, bool ignoreUnitTests = false)
         {
-            var baseTestPath = Path.Combine(Invoker.GeneratorTestsDir, "ProtoTests", testName);
-            var clientPath = Path.Combine(baseTestPath, $"Testing.{testName}");
+            var (effectiveBasePathPart, effectiveTestName) = protoPackageVersion == null 
+                ? ( serviceName, serviceName) 
+                : ($"{serviceName}.{protoPackageVersion}", $"{serviceName}.{protoPackageVersion.Replace('v', 'V')}");
```

Comment:
This won't work if we have v1beta - we'd want V1Beta, not V1beta. Maybe just upper-case `protoPackageVersion`?